### PR TITLE
Feature desktop version

### PR DESCRIPTION
**Pull request by Graciano1997**
In this Project I :

- Did the desktop version of the portfolio. 
-  I fixed the linters errors.
- Updated Readme file.

### CODE REVIEW

File: assets/css/style.css
Comment (KanzaTahreem) [inline assets/css/style.css:1]: - [ ] Good job with the desktop version 👍 However, I noticed that the styles for the mobile version are edited for creating a desktop version 😓 Please note that this is not what we are want. The portfolio project is divided into different milestones and with every milestone, we are making progress, right? ✨⛏ And while working on a desktop version, we are not supposed to remove the styling of mobile version. Please make sure that mobile version looks as given in the Figma template

https://github.com/Graciano1997/portofolio-graciano/assets/106119105/a27b59aa-a545-4c03-8449-490d03d0f756

File: assets/css/style.css [hunk 1]
```diff
@@ -1,111 +1,136 @@
...
-    height: 90%;
-    width: 90%;
+  .headline-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-left: 18%;
+    margin-right: 18%;
+    margin-top: 5%;
```
Comment (KanzaTahreem) [inline assets/css/style.css:69]: - [ ] Good job with the headline section 👍👍. However, from `768px` onwards, I noticed an empty space on right side which is not consistent with the design guidelines 😓 Kindly make sure that the desktop version should be fully responsive. Please fix these issues ✨ 

https://github.com/Graciano1997/portofolio-graciano/assets/106119105/703bf751-db6d-4ce3-a355-69e1858b2e00

File: assets/css/style.css [hunk 2]
```diff
@@ -1,111 +1,136 @@
...
   .indicator {
     width: 48px;
     height: 4px;
     background: #36b37e;
     border-radius: 24px;
     margin-top: -5%;
+    display: none;
   }
 
   .works-section {
```
Comment (KanzaTahreem) [inline assets/css/style.css:128]: - [ ] For the works sections, some of the project cards are hidden and the title is also not centered. Please note that we want to get a design close to the Figma template as much as possible according to the project requirements  Kindly fix this⛏🤓 See the video below for more clarity 👇

https://github.com/Graciano1997/portofolio-graciano/assets/106119105/f445a17f-cc65-4834-a62a-07a608ee48d7

File: assets/css/style.css [hunk 3]
```diff
@@ -331,15 +377,15 @@
   }
 
   .about {
```
Comment (KanzaTahreem) [inline assets/css/style.css:379]: - [ ] Similarly, for the about section, an empty space is present on right side and the technologies are overflowing⚠😓 Kindly fix this as well to fulfil the project requirements as mentioned above 👆

image

- [ ] For frameworks and skills, what do you think of adding some space on left and right side? 😄 Can you please add it to get a design close to the Figma template according to the project requirements? ✨

Your design | Given design
--| ---|
image | image

File: assets/css/style.css [hunk 4]
```diff
@@ -375,25 +434,52 @@
...
+    margin-left: 0;
+    margin-right: 0;
     display: flex;
     flex-direction: column;
     justify-content: center;
-    gap: 20px;
-    background-image: url(../img/imgfoot.svg);
+  }
+
+  .contact-container {
```
Comment (KanzaTahreem) [inline assets/css/style.css:458]: - [ ] Good job so far 👍👍 I noticed that in about section, the input fields are overflowing and the section content is hidden ⚠ Please consider fixing this according to the design guidelines to get a design close to the FIgma template as mentioned in the project requirements

image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Graciano1997 🙋‍♀️

Good job so far! I really appreciate the effort you put in!  **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file

## Required Changes 🛑
_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @Graciano1997 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (2)

## Highlights🎉 🎉 

- [ ] Following Figma design😊👌
- [ ] Use the semantics elements💯
- [ ] Nice design .🚀
- [ ] Use the flexbox to place the items 😃👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**